foundation.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: terra-billing-profile-manager
+  description: |
+    Terra Billing Profile Manager provides an API 
+    to set up and control access to billing within 
+    Terra across cloud platforms.
+  tags:
+    - java
+    - dsp
+    - terra
+    - springboot
+    - broadworkspaces
+  annotations:
+    github.com/project-slug: databiosphere/terra-billing-profile-manager
+spec:
+  type: service
+  lifecycle: production
+  owner: broadworkspaces
+  system: terra
+  dependsOn:
+    - component:sam
+  providesApis:
+    - terra-billing-profile-manager-api
+---
+apiVersion: backstage.io/v1alpha1
+kind: API
```

Comment:
What is the difference between the API and the Component kind? Is this supposed to represent a published client vs. the REST API?

Reply:
Sort of. So backstage the tool itself is very flexible since it's a big open source project it can represent lots of different types of software components. The way I would think of it is that `Component` represents BPM as an executable piece of software that runs somewhere. `API` then expresses that the BPM component exposes a REST API that can be consumed by clients. The backstage UI uses this information to do things like render a preview of the swagger page etc. 

The reason for the delineation is that organizations could theoretically have lots of different types of software components other than just backend rest APIs, there could be gRPC services, things that just read off and publish to message queues etc, other things that don't expose APIs at all etc..